### PR TITLE
[#43] Add payload file path to enhance the github action.

### PR DESCRIPTION
**Pull request by umbertix**
Fixes: https://github.com/slackapi/slack-github-action/issues/43

Increase code coverage.
Updated Readme to have the new input type.
Added example for the new input type usage.

###  Summary

Describe the goal of this PR. Mention any related Issue numbers.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/slack-github-action/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: .github/resources/payload-notification.json
Comment (seratch) [inline .github/resources/payload-notification.json:8]: For better e2e testing, can you add a GitHub variable (say, `${{ github.event_name }}`) in this text?

File: .github/workflows/main.yml
Comment (seratch) [inline .github/workflows/main.yml:98]: For other properties, we use a different naming rule. Can we change this?
```suggestion
        payload-file-path: ./.github/resources/payload-notification.json
```

File: README.md
Comment (seratch) [inline README.md:59]: same as above

File: src/slack-send.js
Comment (seratch) [inline src/slack-send.js:39]: Nice way to tell an error! How about being more specific?
```suggestion
        throw new Error(`The payload-file-path may be incorrect. Failed to load the file: ${payloadFilePath}`);
```

File: test/slack-send-test.js
Comment (seratch) [inline test/slack-send-test.js:65]: Thanks for adding tests! I won't repeat but consistently renaming `payloadFilePath` would be appreciated.

**Review by seratch — CHANGES_REQUESTED** [review]

**Review by seratch — APPROVED** [review]
LGTM; thanks for working on this!

**Comment by seratch** [comment]
@stevengill If this change looks good to you, you can merge this 👍  (let's use "Squash and merge" to squash the commits into one)

**Comment by umbertix** [comment]
I don't have enough permissions to merge. I believe you'll have to do the honors.

**Comment by seratch** [comment]
@umbertix Thanks for being patient here! My last comment was to Steve, another maintainer of this repo. I wanted to have his review before merging this PR.

**Review by stevengill — APPROVED** [review]
Looks Great!!

**Pull request merged**
